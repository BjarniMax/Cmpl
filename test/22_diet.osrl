<?xml version="1.0" encoding="UTF-8"?><?xml-stylesheet type="text/xsl" href="http://www.coin-or.org/OS/stylesheets/OSrL.xslt"?>
<osrl xmlns="os.optimizationservices.org"   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="os.optimizationservices.org http://www.optimizationservices.org/schemas/2.0/OSrL.xsd" >
<general>
<generalStatus type="normal">
</generalStatus>
<serviceName>

Optimization Services Solver
Main Authors: Horand Gassmann, Jun Ma, and Kipp Martin
Distributed under the Common Public License 1.0
OS Version: 2.3.2
Build Date: Mar 15 2011
SVN Version: 4086

</serviceName>
<instanceName>22_diet.cmpl</instanceName>
<solverInvoked>COIN-OR cbc</solverInvoked>
</general>
<job>
<timingInformation numberOfTimes="1">
<time type="elapsedTime" unit="second" category="total" >
6.666999999999999e-3</time>
</timingInformation>
</job>
<optimization numberOfSolutions="1" numberOfVariables="8" numberOfConstraints="4" numberOfObjectives="1">
<solution targetObjectiveIdx="-1">
<status type="optimal">
</status>
<variables >
<values numberOfVar="8">
<var idx="0">2</var>
<var idx="1">8</var>
<var idx="2">2</var>
<var idx="3">2</var>
<var idx="4">9.999999999999998</var>
<var idx="5">10</var>
<var idx="6">10</var>
<var idx="7">2</var>
</values>
</variables>
<objectives >
<values numberOfObj="1">
<obj idx="-1">101.14</obj>
</values>
</objectives>
</solution>
</optimization>
</osrl>
