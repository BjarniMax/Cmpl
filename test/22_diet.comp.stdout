CMPL model generating - running
CMPL model generating - finished
OSSolverServices      - running
---------------------------------------------------------------------------------------------------------
General status:     normal
Problem :           22_diet.cmpl
SolverName          COIN-OR cbc
Nr. of Solutions:   1
---------------------------------------------------------------------------------------------------------
Solution nr.:       1
Objective name :    cost
Objective value :   101.14 (min!)

Variables           
Nr. of variables:   8
Name                 Type            Activity         Lower Bound         Upper Bound            Marginal
---------------------------------------------------------------------------------------------------------
x_BEEF                  I                2.00                2.00               10.00                   -
x_CHK                   I                8.00                2.00               10.00                   -
x_FISH                  I                2.00                2.00               10.00                   -
x_HAM                   I                2.00                2.00               10.00                   -
x_MCH                   I               10.00                2.00               10.00                   -
x_MTL                   I               10.00                2.00               10.00                   -
x_SPG                   I               10.00                2.00               10.00                   -
x_TUR                   I                2.00                2.00               10.00                   -
---------------------------------------------------------------------------------------------------------

Constraints         
Nr. of constraints: 4
Name                 Type            Activity         Lower Bound         Upper Bound            Marginal
---------------------------------------------------------------------------------------------------------
A                       G             1500.00              700.00            Infinity                   -
B1                      G             1330.00              700.00            Infinity                   -
B2                      G              860.00              700.00            Infinity                   -
C                       G              700.00              700.00            Infinity                   -
---------------------------------------------------------------------------------------------------------
OSSolverServices      - finished
